src/bulk/common.ts

```diff
@@ -830,46 +798,14 @@ export class FindOperators {
     return this;
   }
 
-  /** Add a delete one operation to the bulk operation */
-  deleteOne(): BulkOperationBase {
+  /** Specifies the collation for the query condition. */
+  collation(collation: CollationOptions): this {
     if (!this.bulkOperation.s.currentOp) {
       this.bulkOperation.s.currentOp = {};
     }
 
-    // Establish the update command
-    const document = {
-      q: this.bulkOperation.s.currentOp.selector,
-      limit: 1
-    };
-
-    // Clear out current Op
-    this.bulkOperation.s.currentOp = undefined;
-    return this.bulkOperation.addToOperationsList(BatchType.DELETE, document);
-  }
-
-  /** Add a delete many operation to the bulk operation */
-  delete(): BulkOperationBase {
-    if (!this.bulkOperation.s.currentOp) {
-      this.bulkOperation.s.currentOp = {};
-    }
-
-    // Establish the update command
-    const document = {
-      q: this.bulkOperation.s.currentOp.selector,
-      limit: 0
-    };
-
-    // Clear out current Op
-    this.bulkOperation.s.currentOp = undefined;
-    return this.bulkOperation.addToOperationsList(BatchType.DELETE, document);
-  }
-
-  removeOne(): BulkOperationBase {
-    return this.deleteOne();
-  }
-
-  remove(): BulkOperationBase {
-    return this.delete();
+    this.bulkOperation.s.currentOp.collation = collation;
```

Comment:
I don't think this should expand the scope of this PR, but I just wanted to comment on something so you can keep it in mind. We have "builders" for a number of things in the driver, specifically for bulk and cursors. I think it would be really great for readability/maintainability if you were doing them all the same way. The bulk builders need a lot of work, and are confusingly coupled, but I think you could use a `[kBuiltOptions]` approach here (or better yet, wrap `[kBuiltOptions]` in some well-typed methods like `addToCurrentOperation` that you can reuse as a toolkit across all "builders" in the codebase).

Reply:
Agreed that it'd be nice to have a single approach to built options. There's already a new ticket for adding a builder method for `arrayFilters` (NODE-2751), so perhaps we should prioritize this work as part of NODE-2734.